app/internal/page/share_code_test.go

```diff
@@ -0,0 +1,103 @@
+package page
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/notify"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+)
+
+type mockNotifyClient struct {
+	mock.Mock
+}
+
+func (m *mockNotifyClient) TemplateID(id notify.TemplateId) string {
+	return m.Called(id).String(0)
+}
+
+func (m *mockNotifyClient) Email(ctx context.Context, email notify.Email) (string, error) {
+	args := m.Called(ctx, email)
+	return args.String(0), args.Error(1)
+}
+
+func (m *mockNotifyClient) Sms(ctx context.Context, sms notify.Sms) (string, error) {
+	args := m.Called(ctx, sms)
+	return args.String(0), args.Error(1)
+}
```

Comment:
Would you hate this living in `page/mock_test`?

Reply:
not at all